### ISSUE TITLE
Price quote widget search box tabbing doesn't work

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#2002]
**Source**

LF

**Describe the bug**

This is only for proudcity.com, so not as high-priority, but tabbing for the price quote widget search box doesn't work, just keeps you in the text field.

**To Reproduce**
1. Go to proudcity.com
2. Click on price quote search field
3. Try tabbing out

**Expected behavior**

Tabbing to keep progressing

**Screenshots**

![Screenshot 2022-03-25 at 11-17-59 Local government digital solutions - ProudCity](https://user-images.githubusercontent.com/1569463/160178784-d84a6ae9-7f90-404c-b915-cba99f49ac65.png)
